specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01/serverAzureADAdministrators.json

```diff
@@ -1,314 +0,0 @@
-{
```

Comment:
Have you gone through the deprecation process for these APIs?

https://armwiki.azurewebsites.net/api_contracts/APIDeprecationPolicy.html